src/one_dragon/base/notify/push.py

```
@@ -571,26 +572,111 @@ def send_mpnews(self, title, message, media_id, touser="@all"):
+                    if resp and resp.get("errcode") == 0:
+                        self.log_info("企业微信机器人图片推送成功！(无需压缩)")
+                    else:
+                        errcode = resp.get("errcode") if resp else None
+                        errmsg = resp.get("errmsg") if resp else None
+                        self.log_error(
+                            f"企业微信机器人图片推送失败(直发)! status={status}; errcode={errcode}; errmsg={errmsg}; size={orig_size}B; body_snip={body_snip}"
+                        )
+            else:
+                try:
+                    img_bytes_c, _, quality = self._compress_image(image, TARGET_SIZE)
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)